Buildings/Fluid/Air/AirHandlingUnit.mo

```diff
@@ -0,0 +1,258 @@
+within Buildings.Fluid.Air;
+model AirHandlingUnit
+  extends Buildings.Fluid.Air.BaseClasses.PartialAirHandlingUnit(
+    redeclare Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage watVal(
+      final R=R,
+      final delta0=delta0),
+    redeclare Buildings.Fluid.Movers.SpeedControlled_y fan);
+
+  // Parameters for water-side valve
+  parameter Real R=50 "Rangeability, R=50...100 typically"
+  annotation(Dialog(group="Valve"));
+  parameter Real delta0=0.01
+    "Range of significant deviation from equal percentage law"
+    annotation(Dialog(group="Valve"));
+
+  // Parameters for electric heater
+  parameter Modelica.SIunits.Time tauEleHea = 30
+    "Time constant at nominal flow for electric heater (if energyDynamics <> SteadyState)"
+     annotation (Dialog(tab = "Dynamics", group="Electric heater",
+       enable=not (energyDynamics==Modelica.Fluid.Types.Dynamics.SteadyState)));
+  parameter Modelica.SIunits.Efficiency etaHea = 1.0 "Efficiency of electrical heater"
+    annotation (Dialog(group="Electric heater"));
+  parameter Modelica.SIunits.HeatFlowRate QHeaMax_flow(min=0)
+    "Nominal heating capacity of eletric heater,positive"
+    annotation (Dialog(group="Electric heater"));
+  // Parameters for humidifier
+  parameter Modelica.SIunits.MassFlowRate mWatMax_flow(min=0)
+    "Nominal humidification capacity for humidifier, positive for humidification"
+    annotation (Dialog(tab="General",group="Humidifier"));
+  parameter Modelica.SIunits.Temperature THum = 293.15
+    "Temperature of water that is added to the fluid stream by the humidifier"
+    annotation (Dialog(group="Humidifier"));
+  parameter Modelica.SIunits.Time tauHum = 30
+    "Time constant at nominal flow for humidifier(if energyDynamics <> SteadyState)"
+     annotation (Dialog(tab = "Dynamics", group="Humidifier",
+       enable=not (energyDynamics==Modelica.Fluid.Types.Dynamics.SteadyState)));
+
+  // parameters for heater controller
+  parameter Real yValLow(min=0, max=1, unit="1")=0.1
+  "if yVal<=yValLow, hysVal.y switches to false"
```

Comment:
move in the text line to make it being more readable, as well as the annotation text line.